File: L2/PY/casa/7.py
x=input('\n')
match x:
    case '1':
        x='Engenharia'
    case '2':
        x='Edificações'
    case '3':
        x='Sistemas Elétricos'
    case '4':
        x='Turismo'
    case '5':
        x='Análise de Sistemas'
    case _:
        x='Inválido'
print(x,'\n ')

File: 1_year/LGDP/L2/C/casa/7.c
#include<stdio.h>
#include<string.h>
#include<stdlib.h>
//
int main(){
    char*inp=(char*)malloc(3*sizeof(*inp)),*out=(char*)malloc(20*sizeof(*out));
    printf("\nInsira o numero do curso: ");
    scanf("%s",inp);
    if(strcmp(inp,"1")==0)
        out="Engenharia";
    else if(strcmp(inp,"2")==0)
        out="Edificações";
    else if(strcmp(inp,"3")==0)
        out="Sistemas Elétricos";
    else if(strcmp(inp,"4")==0)
        out="Turismo";
    else if(strcmp(inp,"5")==0)
        out="Analise de Sistemas";
    else 
        out="Curso invalido";
    printf("%s\n\n",out);
    return 0;
}